yt_dlp/YoutubeDL.py

```diff
@@ -791,6 +791,14 @@ def _bidi_workaround(self, message):
                       for _ in range(line_count))
         return res[:-len('\n')]
 
+    @staticmethod
+    def _make_channel_prefix(style, name=None):
+        def wrapped(format_function, prefix=name):
+            if prefix is None:
+                raise ValueError('No channel prefix was given')
+            return f'[{format_function(prefix, style)}]'
+        return wrapped
+
```

Comment:
This might be better if it were wrapping whole `Channels` `Namespace` and maybe `Styles` as well. But later would mean redoing a lot already done here. Changing:
```py
self._format_err('payload', self.Styles.STYLE)
```
to
```py
self.Styles.STYLE(self._format_err, 'payload')
```
not sure if this worth it.